config/puma.rb

```diff
@@ -0,0 +1,18 @@
+workers Integer(ENV["WEB_CONCURRENCY"] || 1)
+threads_count = Integer(ENV["MAX_THREADS"] || 5)
+threads threads_count, threads_count
+
+preload_app!
+
+rackup      DefaultRackup
+port        ENV["PORT"]     || 3000
```

Comment:
Unnecessary spacing detected.<br>Put one space between the method name and the first argument.